index.src.html

```diff
@@ -2347,14 +2342,57 @@
             {{AudioData/[[resource reference]]}}.
         2. Let |reference| be a new reference to |resource|.
         3. Assign |reference| to {{AudioData/[[resource reference]]}}.
-        4. Assign the values of |data|'s {{AudioData/[[detached]]}},
+        4. Assign the values of |data|'s {{platform object/[[Detached]]}},
             {{AudioData/[[format]]}}, {{AudioData/[[sample rate]]}},
             {{AudioData/[[number of frames]]}},
             {{AudioData/[[number of channels]]}}, and
             {{AudioData/[[timestamp]]}} slots to the corresponding slots in
             |clone|.
     2. Return |clone|.
 
+: <dfn>Close AudioData</dfn> (with |data|)
+:: Run these steps:
+        1. Assign `true` to |data|'s {{platform object/[[Detached]]}} internal slot.
+        2. Assign `null` to |data|'s {{AudioData/[[resource reference]]}}.
+
+### Transfer and Serialization ###{#audiodata-transfer-serialization}
+
+NOTE: Transferring an {{AudioData}} moves its {{AudioData/[[resource
+    reference]]}} to the destination object, and "closes" the original source
+    object. Serializing an {{AudioData}} creates a "clone" of the source object,
```

Comment:
Instead of using quotes around "closes" and "clone" here, could these be linked to the relevant method or algorithm steps?

Reply:
Done. The new note explicitly references the close() and clone() methods.